src/components/token-screen-content-container/token-screen-content-container.tsx

```diff
@@ -30,15 +30,19 @@ export const TokenScreenContentContainer: FC<Props> = ({ historyComponent, infoC
   return (
     <>
       <View style={styles.headerContainer}>
-        <TouchableOpacity style={styles.delegateContainer} onPress={() => navigate(ScreensEnum.Delegation)}>
-          {isBakerSelected ? (
-            <Text style={styles.delegateText}>Rewards & Redelegate</Text>
-          ) : (
-            <Text style={styles.delegateText}>
-              Delegate: <Text style={styles.apyText}>{delegationApy}% APY</Text>
-            </Text>
-          )}
-        </TouchableOpacity>
+        {segmentedControlIndex === 0 ? (
```

Comment:
the same condition as `showHistoryComponent` value contains.  
Reuse it and make correct name